webrtc.html

```diff
@@ -11830,9 +11830,8 @@ <h2 id="sec.idp-error-handling">IdP Error Handling</h2>
         <tr>
           <td><dfn id="event-track"><code>track</code></dfn></td>
           <td><code><a>RTCTrackEvent</a></code></td>
-          <td>A new incoming <code>MediaStreamTrack</code> has been created,
-          and an associated <code>RTCRtpReceiver</code> has been added to the
-          set of receivers.</td>
+          <td>A new incoming <code>MediaStreamTrack</code> has been created
+          to represent incoming media.</td>
```

Comment:
Since the event includes an `RTCRtpReceiver`, it may still be worth tying this to receivers in some way. Such as "... to represent incoming media received by a specific `RTCRtpReceiver`."